public/global.css

```diff
@@ -40,28 +41,38 @@ input:focus {
 	outline: none;
 }
 
-
-@media screen and (min-width: 450px){
+@media screen and (min-width: 576px) {
+/* @media screen and (min-width: 450px){ */
 	/* extra small screen styles */
 	.xs-inline-block {
 		display: inline-block !important;
 	}
 	.xs-hidden {
 		display: none !important;
 	}
+
+	body {
+		background-color: lawngreen !important;
```

Comment:
I'm very fond of our "color scheme"... I'm just wondering when we should get rid of that?

Reply:
Commented out.